lib/FlavorDownloader.js

```diff
@@ -266,6 +296,12 @@ var FlavorDownloader = (function () {
                     return results;
                 }).then(function (results) {
                     return concludeOperation(results);
+                }).catch(function (err) {
+                    if (err==httpUtils.HttpDownloadNotChanged) {
+                        return Q.resolve();
```

Comment:
I thing its not good idea to write it here, since all the promises from downloadFlavorManifest() until here must not taking care in error.
I would write it like this 

        return Q.fcall(function () {
            return downloadFlavorManifest()
                .then(  #here all the logic ....#,
function(err){
 if (err instanceof  Error)
                    throw new Error("Failed to download file.\nUrl: " + url + "\nDestination: " + pathDestination + "\n" + err + " stack: " + err.stack);
                else {
                    return Q.reject(err);
})